lib/matplotlib/blocking_input.py

```diff
@@ -25,11 +25,10 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+import logging
```

Comment:
not to be a pain but https://github.com/matplotlib/matplotlib/pull/7454#discussion_r87736843 (yes that should probably be documented in the style guide)

Reply:
The only pain is that I don't know what you mean here ;-)  should logging come after six?  You just told me to make everything alphabetical!  ;-)

Reply:
The standard order should be

future imports + six
(empty line)
standard library
(empty line)
third party
(empty line)
matplotlib other modules

(~ https://www.python.org/dev/peps/pep-0008/#imports)

and try to keep things ordered in each section.

Reply:
OK, I didn't clean up all the other imports that don't follow this, but I tried to fix `logging` to follow this...